examples/source/1.components/amp-autocomplete.html

```diff
@@ -96,9 +100,11 @@
 
     A **datasource** must be a JSON object containing an array property `items`, and can be specified inline with a child `script type="application/json"` tag, or a server endpoint specified with a `src` attribute.
   -->
-  <form>
-    <amp-autocomplete filter="substring" min-characters="0">
-      <input type="text">
+  <form class="sample-form" method="post"
+        action-xhr="<% hosts.backend %>/components/amp-form/submit-form-input-text-xhr"
```

Comment:
we're currently deprecating the ampbyexample backend. Can you please change all backend calls to:

`<% hosts.platform %>/documentation/examples/api/echo`

Reply:
The change that resolves the error you are referring to isn't in prod yet, but should be fixed in the next push. Sorry if this change was a bit premature. 😇 

Making the other change now!